examples/kubernetes/access_points/README.md

```diff
@@ -97,4 +97,6 @@ as this could subject you to
     if you configured your access point with `/ap1`, the above would mount to
     `/ap1/my/subpath`.
   - As with normal volume path, the `[Subpath]` must already exist prior to consuming
-    the volume from a pod.
\ No newline at end of file
+    the volume from a pod.
+
+- `awscredsuri` mount option is not supported through efs-csi-driver due to security concerns.
```

Comment:
Not due to security concerns, it is due to this option is designed and used by ECS tasks.